scripts/prices.js

```diff
@@ -0,0 +1,73 @@
+import { getBalance } from './global';
+import { cMarket, fillCurrencySelect } from './settings';
+
+/**
+ * CoinGecko's endpoint for PIVX data, optimised for least bandwidth
+ * - No localisation, tickers, community data, developer data or sparklines
+ */
+export const COINGECKO_ENDPOINT =
```

Comment:
I think we should keep this in chain_params, and if it's null for the current chain we can either remove the price or set it to 0, this way testnet coins don't display mainnet prices

Reply:
I don't think there will ever be an instance where non-mainnet has a price (so "alternative" price tracking or per-network prices wouldn't really be useful).

But for consistency, I think keeping the prices as-is on *all* networks is fine, so tPIV would still show a value to keep the UI consistent for testing purposes, thoughts?

Reply:
I'm still not convinced about this, tPIV should be valued at 0$, but it seems like i'm alone on this, since it's a small detail I think we can merge